packages/volto/news/5284.feature

```diff
@@ -0,0 +1 @@
+Shows working copy options for Plone Site. @wesleybl
```

Comment:
```suggestion
Show the working copy actions (checkin/checkout) based on whether they are enabled in the backend, instead of the `hasWorkingCopySupport` setting. @wesleybl, @davisagli
```